tests/Facility.CodeGen.CSharp.UnitTests/CSharpGeneratorTests.cs

```diff
@@ -79,6 +79,96 @@ public void GenerateEnumStringConstants()
 		StringAssert.Contains("public const string Maybe = \"maybe\";", file.Text);
 	}
 
+	[Test]
+	public void GenerateExternalDtoPropertyWithNamespace()
+	{
+		const string definition = "[csharp] service TestApi { [csharp(name: \"ExternThingDto\", namespace: \"Some.Name.Space\")] extern data Thing; data Test { thing: Thing; } }";
```

Comment:
```suggestion
		const string definition = "service TestApi { [csharp(name: \"ExternThingDto\", namespace: \"Some.Name.Space\")] extern data Thing; data Test { thing: Thing; } }";
```

I don't know why `[csharp]` is included in `GenerateEnumStringConstants`, but I don't think it is necessary or useful.

Reply:
Good call, removed 3fea02e1